packages/dx-react-scheduler-material-ui/src/templates/appointment-form/recurrence/layouts/daily.jsx

```diff
@@ -36,7 +37,7 @@ export const Daily = ({
       textEditorComponent={textEditorComponent}
       labelComponent={labelComponent}
       changeRecurrenceInterval={changeRecurrenceInterval}
-      interval={recurrenceOptions.interval}
+      interval={getRecurrenceInterval(recurrenceOptions.interval)}
```

Comment:
Why are you validate `interval` here instead of validating it in `defaultProps`?